packages/core/components/FileRow/Cell.tsx

```diff
@@ -94,37 +96,39 @@ export default class Cell extends React.Component<React.PropsWithChildren<CellPr
         const { className, width } = this.props;
         const { containerClassName, provisionalWidth, resizeTargetClassName } = this.state;
         return (
-            <div
-                ref={this.cell}
-                className={classNames(styles.resizableCell, containerClassName, className)}
-                onContextMenu={this.props.onContextMenu}
-                onDoubleClick={this.onDoubleClick}
-                title={this.props.title}
-                style={{
-                    width: `${(provisionalWidth || width) * 100}%`,
-                    minWidth: Cell.MINIMUM_WIDTH,
-                }}
-            >
-                {this.props.children}
-                <span
-                    className={classNames(styles.resizeTarget, resizeTargetClassName)}
-                    ref={this.resizeTarget}
-                />
-            </div>
+            <Tooltip content={this.props.title}>
+                <div
+                    ref={this.cell}
+                    className={classNames(styles.resizableCell, containerClassName, className)}
+                    onContextMenu={this.props.onContextMenu}
+                    onDoubleClick={this.onDoubleClick}
+                    style={{
+                        width: `${(provisionalWidth || width) * 100}%`,
+                        minWidth: Cell.MINIMUM_WIDTH,
+                    }}
+                >
+                    {this.props.children}
+                    <span
+                        className={classNames(styles.resizeTarget, resizeTargetClassName)}
+                        ref={this.resizeTarget}
+                    />
+                </div>
+            </Tooltip>
```

Comment:
<img width="994" alt="Screenshot 2024-09-18 at 10 54 33 AM" src="https://github.com/user-attachments/assets/cca4f70d-ef5d-45c9-93e9-ed0c90ae1357">
Possibly affecting styling here

Reply:
Thought this had gone away. Fixed here https://github.com/AllenInstitute/biofile-finder/pull/241/commits/ddc96fa7065f40e713d87c5c000f779acb642104